cfme/services/workloads.py

```diff
@@ -2,28 +2,79 @@
 """ A model of Workloads page in CFME
 """
 from functools import partial
-from cfme.web_ui import accordion, menu
 
-vm_instances_tree = partial(accordion.tree, "VMs & Instances")
-templates_images_tree = partial(accordion.tree, "Templates & Images")
+from navmazing import NavigateToSibling, NavigateToAttribute  # NOQA
+from cfme.exceptions import DestinationNotFound
+from cfme.web_ui import accordion, match_location, toolbar
+from utils.appliance import Navigatable
+from utils.appliance.implementations.ui import navigator, CFMENavigateStep
 
+vm_instances_tree = partial(accordion.tree, "VMs & Instances", 'All VMs & Instances')
+templates_images_tree = partial(accordion.tree, "Templates & Images", 'All Templates & Images')
 
-@menu.extend_nav
-class services_workloads:
-    def service_vms_instances(_):
-        vm_instances_tree("All VMs & Instances")
+QUADICON_TITLE_LOCATOR = ("//div[@id='quadicon']/../../../tr/td/a[contains(@href,'vm_infra/x_show')"
```

Comment:
I don't think this is needed? Not used in your code here at all, and the only other place I see it in the codebase is in cfme.infrastructure.virtual_machines